sql/test/TPC-h/test_tpch.sh

```diff
@@ -15,7 +15,7 @@ rm -f /tmp/nation_count
 START=$SECONDS
 ./makeSqliveDb.sh
 TOTAL=$(($SECONDS - $START))
-if (( TOTAL > 180 ));
+if (( TOTAL > 1800 ));
```

Comment:
It seems that load-csv was a lot faster

Reply:
Am I reading this right as a ~10x slowdown going from (a) sqlite CSV output piped into `SKDB load-csv` to (b) sqlite SQL-dump output piped into `SKDB`?

That makes sense to me -- when we're dealing with csv input we don't do any (sql) parsing, batch those writes into reasonable chunks, etc.

But if the test is taking 30 minutes now, that seems like an issue.  We could pipe more tables through `head` to cut down the input sizes as a bandaid fix I suppose, and revisit once we do some more work on batch insertions?

Reply:
On my machine the actual time I see is 1334, but I didn't go back and check what a recent time with load-csv was. I did run this test before and had it pass though, so it still is roughly in the region of 10x. This test is not run by default though, it takes far more memory than the CI machines have.